### PR TITLE
hwdb: add a vmbus id for HyperV Video device

### PR DESCRIPTION
**Pull request by Fearyncess**
This PR will add a vmbus id. It will fix a problem that no DM can be started in a HyperV VM sometimes.

### CODE REVIEW

File: hwdb.d/60-seat.hwdb [hunk 1]
```diff
@@ -28,6 +28,13 @@ fb:pci:v000018CAd00000020*
 fb:pci:v00001414d00005353*
  ID_TAG_MASTER_OF_SEAT=1
 
+# In some HyperV VMs, the video device only can be recognized by a vmbus id.
+# So we should add a vmbus id for HyperV Video device.
+#
+# This id is extracted from Linux kernel's hyperv.h
+fb:vmbus:02780ada77e3ac4a8e770558eb1073f8
```
Comment (yuwata) [inline hwdb.d/60-seat.hwdb:35]: Let me confirm that vmbus id is not prefixed with `v`, unlike pci cases, right?

Comment (lunzima) [reply hwdb.d/60-seat.hwdb:35]: I am working with the committer to investigate the issue with Hyper-V clients not displaying properly.
I used this patch to get the correct behavior. Without this patch, sddm and lightdm do not display the interface properly.
The device id here is 
`fb:vmbus:02780ada77e3ac4a8e770558eb1073f8`

File: hwdb.d/parse_hwdb.py [hunk 2]
```diff
@@ -74,7 +74,7 @@
 # Those patterns are used in type-specific matches
 TYPES = {'mouse':    ('usb', 'bluetooth', 'ps2', '*'),
          'evdev':    ('name', 'atkbd', 'input'),
-         'fb':       ('pci'),
+         'fb':       ('pci','vmbus'),
```
Comment (YHNdnzj) [inline hwdb.d/parse_hwdb.py]:
```suggestion
         'fb':       ('pci', 'vmbus'),
```

Comment (Fearyncess) [reply hwdb.d/parse_hwdb.py]: formatted

File: hwdb.d/60-seat.hwdb (hunk 1 above)
Comment (Fearyncess) [reply hwdb.d/60-seat.hwdb:35]: yep, according the `udevadm info` output and the data in `hwdb.d/20-vmbus-class.hwdb`, it doesn't have a `v` prefix.

**Review by bluca — APPROVED** [review]

File: hwdb.d/60-seat.hwdb (hunk 1 above)
Comment (yuwata) [reply hwdb.d/60-seat.hwdb:35]: Thanks.

**Review by yuwata — APPROVED** [review]

**Pull request merged**